### ISSUE TITLE
Total Freeze (MO)

### ISSUE THREAD

**Issue by Rinelw** [ousnius/BodySlide-and-Outfit-Studio#332]
**Description**
Whenever I run batch build and I have an option to choose output everything except of a mouse cursor freezes after the cursor leaves the batch build window boundary. PC Restart is the only option to unfreeze

**How to reproduce**
1. Start BodySlide
2. Select your presets
3. Click on 'Batch Build'
4. Click 'Ok' 
5. Move your cursor outside the batch build window area.
6. See the error.

**Personal Notes**
I Believe there might be caused either by combination of MO and having many outfits, or just having many outfits, because before it didn't happen to me.
The bug also appears when you commence the batch build but in that case, it unfreezes after it processes some of the outfits.

**Comment by ousnius** [comment]
Multiple people have reported this and this happens with MO only. Not sure what I would do here, nor how to debug it or what could possibly cause it. The MO developers might be better off looking at it first.

**Comment by Al12rs** [comment]
@aerd2 Could you drop by the issue linked above and help us diagnose this issue?

**Comment by ousnius** [comment]
Closing the issue on this side (not resolved, though).

**Comment by HeyVasyan** [comment]
TL;DR
This has nothing to do with BodySlide, nor with MO. It's just yet another case of Windows shooting itself in the foot.

TS;WM
I ran into the same issue and tried to build my files without MO as was suggested here, but to no avail. So, after a debugging session, I was surprised to find that freezes happen after a WinAPI call (setScrollInfo) deep inside wxWidgets when it creates a "Choose output set" window with a lot of sets (500+ in my case). Doesn't seem to make any sense? Turns out DWM in Windows 10 chokes on scrollable windows with many children, causing the entire windows gui to freeze (source: https://stackoverflow.com/questions/59655147/why-is-my-mfc-application-hanging-after-interacting-with-both-scroll-bars/60063806). Don't know if this helps nor if the issue still happens at all with newer Windows builds, but the provided solution works like a charm for me on v1909.
I.e.
1) Download the appropriate ADK version from https://docs.microsoft.com/en-us/windows-hardware/get-started/adk-install
2) Install just the Compatibility Administrator tool, you don't need anything else
3) Create a new database with a ScrollWindowsExFlags fix for BodySlide
4) Install the database
5) Enjoy freezeless BodySlide👍
